services/business/checkers/route_rule_checker.go

```diff
@@ -4,11 +4,14 @@ import (
 	"github.com/kiali/kiali/kubernetes"
 	"github.com/kiali/kiali/services/business/checkers/route_rules"
 	"github.com/kiali/kiali/services/models"
+	"k8s.io/api/core/v1"
```

Comment:
Bad import order, it should be
standard imports
LINE
third pary
LINE
kiali imports

If not @jmazzitelli will chase you :-)

Reply:
Upsie! Just forgot that rule here.